src/main/java/org/traccar/protocol/MictrackLowAltitudeFlightProtocol.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2015 - 2018 Anton Tananaev (anton@traccar.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.traccar.protocol;
+
+import io.netty.handler.codec.string.StringDecoder;
+import io.netty.handler.codec.string.StringEncoder;
+import org.traccar.BaseProtocol;
+import org.traccar.PipelineBuilder;
+import org.traccar.TrackerServer;
+
+public class MictrackLowAltitudeFlightProtocol extends BaseProtocol {
+
+    public MictrackLowAltitudeFlightProtocol() {
+        addServer(new TrackerServer(false, getName()) {
+            @Override
+            protected void addProtocolHandlers(PipelineBuilder pipeline) {
+                pipeline.addLast(new StringDecoder());
```

Comment:
Frame decoder is missing.

Reply:
I am not quite sure what kind of frame decoder we would need here. One message contains one time the device identifier (IMEI) and then multiple positions, i.e. DeviceID$PosA$PosB$PosC.

Can you explain it a little more?

Reply:
Any TCP based protocol needs a frame decoder.